tasks/src/tabular-regression/about.md

```diff
@@ -3,7 +3,19 @@
 Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species and length.
 
 ## Use Cases
+Whenever we have a set of variables recorded in a table and using them we try to predict a target variable, this kind of scenario could come under tabular regression. Some of the popular tabular use-cases are:
 
+- **Predicting a continuous target variable**: 
+  - It is used in case when we have to predict a continuous variable based on a set of input variable(s).
+  - For example, say we want to predict the `sales` of an Ice-Cream shop based on `temperature` and `hours` shop was open. Here we can build a regression model with `temperature` and `hours` as input variable and `sales` as target variable.
+
+- **Filling Missing Values**:
+  - In real-world due to human error or other reasons some of the value can be missing or there might be some problem due to which no data was recorded. 
+  - Consider the example above, say the shopkeeper watch was broken and they forgot to calculate the `hours` for which the shop was open. This will lead to a missing value in their dataset.
```

Comment:
```suggestion
  - Considering the example above, say the shopkeeper's watch was broken and they forgot to calculate the `hours` for which the shop was open. This will lead to a missing value in their dataset.
```